squire_core/src/state.rs

```diff
@@ -149,6 +147,29 @@ impl ServerState for AppState {
         todo!()
     }
 
+    async fn get_tourn_summaries(
+        &self,
+        including: RangeInclusive<usize>,
+    ) -> Vec<TournamentSummary> {
+        if let Ok(cursor) = self
+            .get_tourns()
+            .find(
+                None,
+                FindOptions::builder().sort(doc! {"$natural":-1}).build(),
```

Comment:
I'm not sure of the internals of the options builder. Do we know the performance cost of building them? Would it make sense to use something like a [`Lazy`](https://docs.rs/once_cell/latest/once_cell/sync/struct.Lazy.html)?

Reply:
I found a simple benchmark here for the builder crate that the mongodb rust driver uses: https://www.reddit.com/r/rust/comments/74ic2i/comment/do09tud/

It seems like the builder has little overhead compared to manually constructing the object, which we would need to do with a static version anyway, since `FindOptions` is not `Copy`.

Reply:
Sounds good. Using something like `Lazy` is pretty common with regex since compiling the regex is often more involved than executing it.